aip/general/0164.md

```diff
@@ -0,0 +1,155 @@
+---
+id: 164
+state: reviewing
+created: 2020-10-06
+placement:
+  category: design-patterns
+  order: 95
+---
+
+# Soft delete
+
+There are several reasons why a client could desire soft delete and undelete
+functionality, but one over-arching reason stands out: recovery from mistakes.
+A service that supports undelete makes it possible for users to recover
+resources that were deleted by accident.
+
+## Guidance
+
+APIs **may** support the ability to "undelete", to allow for situations where
+users mistakenly delete resources and need the ability to recover.
+
+If a resource needs to support undelete, the Delete method **must** simply mark
```

Comment:
Put Delete in backticks, similar to `List` and `Create` earlier for the `parent` field?

Reply:
Done.